##### TAG DEFINITION OF ENVIRONMENT #####
# [ID, SIZE(m), X(m), Y(m), Z(m), psi(Deg)]
TAG_TRUE_RT:
  TAGS:
    - [256, 0.16, 1.00, 0.00, 1.000, 0.0]
    - [257, 0.16, 0.25, 0.75, 1.000, 90.0]
    - [258, 0.16, 0.50, 0.75, 1.000, 90.0]